README.md

```diff
@@ -1,29 +1,63 @@
 # Evergreen.py
 
 A client library for the Evergreen API written in python. Currently supports the V2 version of
-the API. For more details, see https://github.com/evergreen-ci/evergreen/wiki/REST-V2-Usage .
+the API. For more details, see https://github.com/evergreen-ci/evergreen/wiki/REST-V2-Usage.
 
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/evergreen.py) [![PyPI](https://img.shields.io/pypi/v/evergreen.py.svg)](https://pypi.org/project/evergreen.py/) [![Coverage Status](https://coveralls.io/repos/github/evergreen-ci/evergreen.py/badge.svg?branch=master)](https://coveralls.io/github/evergreen-ci/evergreen.py?branch=master)
 
-## Documentation
+## Table of contents
 
-You can find the documentation [here](https://evergreen-ci.github.io/evergreen.py/).
+1. [Description](#description)
+2. [Dependencies](#dependencies)
+3. [Installation](#installation)
+4. [Usage](#usage)
+5. [Documentation](#documentation)
+6. [Contributor's Guide](#contributors-guide)
+    - [High Level Architecture](#high-level-architecture)
+    - [Setting up a local development environment](#setting-up-a-local-development-environment)
+    - [linting/formatting](#lintingformatting)
+    - [Running tests](#running-tests)
+    - [Automatically running checks on commit](#automatically-running-checks-on-commit)
+    - [Staging Environment](#staging-environment)
+    - [Versioning](#versioning)
+    - [Code Review](#code-review)
+    - [Deployment](#deployment)
+    - [Additional Documentation](#additional-documentation)
+7. [Resources](#resources)
 
-## Usage
+## Description
 
-```
->>> from evergreen.api import EvgAuth, EvergreenApi
->>> api = EvergreenApi.get_api(EvgAuth('david.bradford', '***'))
->>> project = api.project_by_id('mongodb-mongo-master')
->>> project.display_name
-'MongoDB (master)'
+This is a Python client library for interacting with Evergreen and Evergreen objects. It currently only
+supports the V2 version of Evergreen's api. It can be used either by Python code in a separate application
+or on the command line to get data about Evergreen objects quickly and easily. 
+
+
+## Dependencies
+
+* Python 3.7 or later
+* Poetry 1.1 or later
+
+## Installation
+
+```bash
+$ poetry install
```

Comment:
Again, this is meant for users, so we probably want to use `pip install evergreen.py` instead here.